fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/PlanOrderingKey.java

```diff
@@ -65,13 +68,24 @@ public class PlanOrderingKey {
     private final int prefixSize;
     private final int primaryKeyStart;
     private final int primaryKeyTail;
+    @Nonnull
+    private final Set<Integer> equalityBound;
```

Comment:
I believe these are _later duplicates_ of equality bound keys. Perhaps the name could reflect that.